docs/kqueen.cli_gcloud_examples.rst

```diff
@@ -0,0 +1,139 @@
+
+.. code-block:: bash
+
+   $ TOKEN=$(curl -s -H "Content-Type: application/json" --data '{ "username": "testusername", "password": "testpassword" }' -X POST 127.0.0.1:5000/api/v1/auth | jq -r '.access_token')
+   $ echo $TOKEN
+   $ USER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/users | jq -r '.[] | select (.username == "testusername").id')
+   $ echo $USER_ID
+   $ SERVICE_ACCOUNT_INFO=$(cat ../key.json)
+   $ echo $SERVICE_ACCOUNT_INFO
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testprovisioner\", \"engine\": \"kqueen.engines.GceEngine\", \"owner\": \"User:$USER_ID\", \"parameters\": { \"project\": \"$GCE_PROJECT_ID\", \"service_account_info\": $SERVICE_ACCOUNT_INFO } }" -X POST 127.0.0.1:5000/api/v1/provisioners | jq
+
+Deploy Kubernetes Cluster using Kqueen with GKE provisioner:
+
+.. code-block:: bash
+
+   $ PROVISIONER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/provisioners | jq -r '.[] | select (.name == "testprovisioner").id')
+   $ echo $PROVISIONER_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" --data "{ \"name\": \"testcluster\", \"owner\": \"User:$USER_ID\", \"provisioner\": \"Provisioner:$PROVISIONER_ID\", \"metadata\": { \"machine_type\": \"n1-standard-1\", \"node_count\": 1, \"zone\": \"us-central1-a\" } }" -X POST 127.0.0.1:5000/api/v1/clusters | jq
+
+Check the status of the cluster by query Kqueen API (run this command multiple times):
+
+.. code-block:: bash
+
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/clusters
+
+Download kubeconfig from kqueen "testcluster":
+
+.. code-block:: bash
+
+   $ CLUSTER_ID=$(curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/clusters | jq -r '.[] | select (.name == "testcluster").id')
+   $ echo $CLUSTER_ID
+   $ curl -s -H "Authorization: Bearer $TOKEN" -H "Content-Type: application/json" 127.0.0.1:5000/api/v1/clusters/$CLUSTER_ID/kubeconfig > kubeconfig.conf
+
+Use kubeconfig and check kubernetes:
+
+.. code-block:: bash
+
+   $ export KUBECONFIG=$PWD/kubeconfig.conf
+   $ kubectl get nodes
+   $ kubectl get componentstatuses
+   $ kubectl get namespaces
+
+Install helm to install application easily:
+
+.. code-block:: bash
+
+   $ curl -s $(curl -s https://github.com/kubernetes/helm | awk -F \" "/linux-amd64/ { print \$2 }") | tar xvzf - -C /tmp/ linux-amd64/helm
+   $ sudo mv /tmp/linux-amd64/helm /usr/local/bin/
+   $ kubectl create serviceaccount tiller --namespace kube-system
+   $ kubectl create clusterrolebinding tiller-cluster-rule --clusterrole=cluster-admin --serviceaccount=kube-system:tiller
+   $ helm init --service-account tiller
+   $ sleep 30
+   $ helm repo update
+
+Install few apps to utilize the cluster:
```

Comment:
To remove a cluster?

Please add an intro. Smth like:
You can easily remove a cluster using the ... application...